tests/integration/test.js

```diff
@@ -190,3 +188,51 @@ describe('Mugshot integration with disabled acceptFirstBaseline', function() {
     return wdioInstance.end();
   });
 });
+
+
+describe('Mugshot integration with recursive rootDirectory', function() {
```

Comment:
It differs because I have to give Mugshot the directory that `tmp` created. I'll create a new module.